runio.js

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const path = require('path');
+const { stopOnFail, chdir, git, copy, exec, replaceInFile, npmRun, npx, writeToFile, runio } = require('runio');
+
+stopOnFail();
+
+module.exports = {
+  async docs() {
+    await Promise.all([
+      this.docsHelpers(),
+      this.docsPlugins(),
+      this.docsExternalHelpers(),
+    ]);
+
+  },
+
+  async docsPlugins() {
+    // generate documentation for plugins
+    await npx('documentation build lib/plugin/*.js -o docs/plugins.md -f md --shallow --markdown-toc=false --sort-order=alpha');
+    await replaceInFile('docs/plugins.md', cfg => {
+      cfg.replace(/^/,'---\nid: plugins\ntitle: Plugins\n---\n\n')
+    })
+  },
+
+  async docsExternalHelpers() {
+    // generate documentation for helpers outside of main repo
+    console.log('Building @codecepjs/detox helper docs');
+    let helper = 'Detox';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['helper' is never reassigned. Use 'const' instead. (prefer-const)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4496293)